pkg/vere/main.c

```diff
@@ -2780,43 +2780,83 @@ _cw_vile(c3_i argc, c3_c* argv[])
 static void
 _cw_boot(c3_i argc, c3_c* argv[])
 {
-  if ( 8 > argc ) {
+  if ( 18 > argc ) {
```

Comment:
This conditional can be removed, we have real argument parsing and validation now.